tests/metrics/test_pd_extractors.py

```diff
@@ -19,6 +20,51 @@
 from fklearn.validation.validator import validator
 
 
+def test__split_evaluator_extractor_iteration__default():
+    logs = {
+        'split_evaluator__split_0': {'roc_auc': 0.48},
+        'split_evaluator__split_1': {'roc_auc': 0.52},
+    }
```

Comment:
As this `logs` var is being used in more than one test would be cool to transform it into a fixture instead of repeat the definition, same for the named logs in `test__split_evaluator_extractor_iteration__with_eval_name` test

Reply:
Also most of these tests are very similar, would be nice to use the pytest parametrize to just change the parameters. [Example](https://github.com/nubank/fklearn/blob/master/tests/training/test_pipeline.py#L14)

Reply:
done